src/components/modals/UniqueIdentifierModal.js

```diff
@@ -34,8 +35,11 @@ export class UniqueIdentifierModal extends Component<Props> {
       uniqueIdentifierChanged,
       keyboardType
     } = this.props
-    const type = getUniqueIdentifierType(currencyCode)
-    const description = getUniqueIdentifierDescription(type)
+    let type = ''
+    if (getSpecialCurrencyInfo(currencyCode).uniqueIdentifier) {
+      type = getSpecialCurrencyInfo(currencyCode).uniqueIdentifier.identifierName
+    }
+    const description = sprintf(s.strings.unique_identifier_modal_description, type)
     const title = type
     const label = type
     const confirm = s.strings.unique_identifier_modal_confirm
```

Comment:
Not super critical but `const title = type` and `const label = type` are redundant. I realize that wasn't part of the new changes though.